lale/lib/logisticaix360/Logistic_aix360.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pandas as pd
+from aix360.algorithms.rbm import FeatureBinarizer, LogisticRuleRegression
+
+import lale.docstrings
+import lale.operators
+
+from .util import (
+    _BaseInEstimatorImpl,
+    _categorical_fairness_properties,
+    _categorical_input_predict_proba_schema,
+    _categorical_input_predict_schema,
+    _categorical_output_predict_proba_schema,
+    _categorical_output_predict_schema,
+    _categorical_supervised_input_fit_schema,
+    dataset_to_pandas,
+)
+
+
+class _logisticaix360Impl(_BaseInEstimatorImpl):
+    def __init__(
+        self,
+        *,
+        favorable_labels,
+        protected_attributes,
+        redact=True,
+        preparation=None,
```

Comment:
I have removed the preparation argument ,can you please elaborate me about this subpipeline.

Reply:
Removing the `preparation` argument is all that's needed. In some of our usage examples, we use the `preparation` argument to pass a subpipeline to a fairness mitigator. But since this operator is not a fairness mitigator, I don't think there is a need for the argument.